plotting.py

```diff
@@ -379,11 +379,15 @@ def plot_timehistory_at_height(datasets,
         ax.yaxis.grid()
     
     # Format time axis
-    axs[-1].xaxis.set_minor_locator(mdates.HourLocator(byhour=range(24),interval=6))
-    axs[-1].xaxis.set_minor_formatter(mdates.DateFormatter('%H%M'))
-    axs[-1].xaxis.set_major_locator(mdates.DayLocator())
-    axs[-1].xaxis.set_major_formatter(mdates.DateFormatter('\n%Y-%m-%d'))
-    axs[-1].set_xlabel(r'UTC time')
+    if isinstance(timevalues, (pd.DatetimeIndex, pd.TimedeltaIndex)):
```

Comment:
Did you test whether it works with a TimedeltaIndex as well?

Reply:
Handled in commit 8a39dc2